src/openapi.yaml

```diff
@@ -2558,18 +2550,18 @@ components:
         raId: 21
 tags:
   - name: editor
-    description: used by Protocol Editor SPA
+    description: wip editor endpoints are used by Protocol Editor SPA
```

Comment:
confirm pls

Reply:
Issue #11 relates to how tags should be used. OAS tooling seems to use them for sorting/filtering so it makes sense to me that they be data-oriented (like the petstore example), not devel oriented (existing tags). Therefore in branch `tag-rework` I have pushed a commit removing existing tags and added basic replacements. You will need to remove existing tags and add new ones.

Subject to change in the future (or suggestions), let's use a tagging strategy that multiple tags are OK but they need not be exhaustive. For example a Protocol can always contain Promises, so anything with a `Protocol` tag might also have `Promise`. Let's only use `Promise` if an endpoint operates directly on it. Use the most general tag applicable.

There are proposals for tag nesting, so future OAS versions may have more flexibility.